CHANGELOG.md

```diff
@@ -3,10 +3,19 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.4.8]
+
+### Changed
+
+- Now respects `account_linking=true` in addition to `prompt=select_account` in order to
```

Comment:
can we call this verification? ;-)

Reply:
changed; also updating it in Mozillians  (https://github.com/mozilla/mozillians/pull/2541)